setup.py

```diff
@@ -119,6 +119,7 @@ def is_requirement(line):
         'Framework :: Django',
         'Framework :: Django :: 3.2',
         'Framework :: Django :: 4.0',
```

Comment:
We probably want to remove 4.0 from the list.

Reply:
I have removed this.